jenkins/data-prepper/release-data-prepper-all-artifacts.jenkinsfile

```diff
@@ -4,105 +4,147 @@ pipeline {
     options {
         timeout(time: 1, unit: 'HOURS')
     }
-    agent {
-        docker {
-            label 'Jenkins-Agent-al2-x64-c54xlarge-Docker-Host'
-            image 'opensearchstaging/ci-runner:ci-runner-ubuntu2004-data-prepper-pipeline-v1'
-            alwaysPull true
-            args '-u root'
-        }
-    }
```

Comment:
I probably forget to mention this part still need an agent or it will fall back to main node.
Just restore the section and change to `alpine:3` and remove args.

Thanks.